zeus/optimizer/batch_size/server/database/repository.py

```diff
@@ -0,0 +1,6 @@
+from sqlalchemy.ext.asyncio.session import AsyncSession
```

Comment:
These imports should be done like the following, because the class `AsyncSession` is not really used during runtime. It's just imported for type annotation.

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from sqlalchemy.ext.asyncio.session import AsyncSession
```